INSTALL.md

```diff
@@ -127,3 +127,19 @@ The outputs will be:
 - Statistics JSON file `outPrefix.statistics.json`
 - Statistics tab-separated file: `outPrefix.statistics.tsv`
   - Removing the `--run_statistics` flag will skip the generation of `outPrefix.statistics.*` files
+
+
+## Setting up Python on Amazon Linux 2
+
+Install these beforehand:
+- openssl-devel
+   - sudo yum install openssl-devel
+- rustup
+   - curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+   - source ~/.bashrc
+   - rustc --version
+
+Install Anaconda (current version as of Oct 18, 2023)
+- wget https://repo.anaconda.com/archive/Anaconda3-2023.09-0-Linux-x86_64.sh
+- bash Anaconda3-2023.09-0-Linux-x86_64.sh 
+- source ~/.bashrc
```

Comment:
This is a duplicate of https://github.com/bystrogenomics/bystro/blob/master/INSTALL.md#installing-the-bystro-python-libraries-and-cli-tools. Please take a look at that section and make it more discoverable, so that our next dev has an easier time finding it, it's a bit buried.

openssl-devel is also a dependency in our existing install script https://github.com/bystrogenomics/bystro/blob/master/install/install-rpm-deps.sh